README.md

```diff
@@ -170,8 +170,22 @@ The second one is set per individual HTTP requests by HTTP client. Its default v
 Flink's current implementation of `AsyncTableFunction` does not allow specifying custom logic for handling Flink AsyncIO timeouts as it is for Java API.
 Because of that, if AsyncIO timer passes, Flink will throw TimeoutException which will cause job restart.
 
-The HTTP request timeouts on the other hand will not cause Job restart. In that case, exception will be logged into application logs.
-To avoid job restart on timeouts caused by Lookup queries, the value of `gid.connector.http.source.lookup.request.timeout` should be smaller than `table.exec.async-lookup.timeout`.
+#### Retries
+Lookup source handles auto-retries for two scenarios:
+1. IOException occurs (e.g. timeout)
+2. HTTP server returns response with code, which is marked as temporal error. These codes are defined in table configuration.
+Retries are executed silently, without job restart. After reaching max retries attempts (per request) function will fail and restart job.
```

Comment:
nit:
without job restart -> without restarting the job
function -> operation (maybe)